backend/src/install/mod.rs

```diff
@@ -593,6 +603,42 @@ pub async fn uninstall_impl(ctx: RpcContext, id: PackageId) -> Result<WithRevisi
     })
 }
 
+#[command(rename = "delete-recovered", display(display_none))]
+pub async fn delete_recovered(
```

Comment:
Highest risk of having an error, I don't know if there are resources that need to be removed that shouldn't or aren't here. Wondering if there is a way of ensuring this?